mhs/mhs_common/workflow/sync_async.py

```diff
@@ -18,8 +17,9 @@
 class SyncAsyncWorkflow(common_synchronous.CommonSynchronousWorkflow):
     """Handles the workflow for the sync-async messaging pattern."""
 
+    # TODO: This used to take an outbound transmission object
```

Comment:
As discussed this workflow was reliant on receiving an outbound transmitter, since this was defined in the `Outbound` package it felt odd to be importing the outbound module into the common. This has now been updated so there is a transmission interface in the `integration_adaptors_common.comms` package, the `OutboundTransmission` class implements this and the `SyncAsyncWorkflow` now expects some implementation of this transmission interface